sources/manager.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sources
+
+import (
+	"time"
+
+	. "k8s.io/heapster/types"
+)
+
+type SourceManager interface {
+	ScrapeSources(start, end time.Time) *DataBatch
+}
+
+func NewSourceManager() (SourceManager, error) {
+	return &realSourceManager{}, nil
+}
+
+type realSourceManager struct {}
+
+func (m *realSourceManager) ScrapeSources(start, end time.Time) *DataBatch {
```

Comment:
it should return a list of data batches. Otherwise secifying start doesn't make any sense.